### PR TITLE
add a generic K8s install guide

### PR DESCRIPTION
**Pull request by samos123**
This shows that KubeAI can easily be installed on any K8s cluster.

### CODE REVIEW

File: docs/installation/k8s.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Install on any Kubernetes Cluster
```
Comment (nstogner) [inline docs/installation/k8s.md:1]: Can we rename this to `custom.md` / `any.md` or something? All of the installs are technically k8s.

File: docs/how-to/install-models.md [hunk 2]
```diff
@@ -54,10 +54,52 @@ kubectl explain models.spec
 kubectl explain models.spec.engine
 ```
 
+You can view all example manifets on the [GitHub repository](https://github.com/substratusai/kubeai/tree/main/manifests/models).
```
Comment (nstogner) [inline docs/how-to/install-models.md:57]: `manifets` spelling

Comment (samos123) [reply docs/how-to/install-models.md]: done

File: docs/installation/k8s.md (hunk 1 above)
Comment (samos123) [reply docs/installation/k8s.md]: done, I switched to `any.md`

**Pull request merged**